### PR TITLE
Create new note

### LINKED ISSUE #3: Create new note UI

### PR DESCRIPTION
**Pull request by naufil50**
Fixes #3 

Screenshot 2022-07-28 at 5 04 03 PM

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 1]
```diff
@@ -40,13 +33,28 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
...
               required
             />
-            <Textarea
+            <Input
               label="Description"
               name="description"
               className="w-full flex-grow-0"
-              rows={8}
+              placeholder="Add Description"
+              required
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:44]: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Declaring%20boolean%20props

**Review by yedhink — COMMENTED** [review]
@naufil50 Please go through https://handbook.neetokb.com/articles/git-and-github-essentials#creating-pull-request and update all your PR's title's.

Also added one comment.

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 1 above)
Comment (naufil50) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:44]: I guess this is correct as per book. have just added boolean variable.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:44]: @naufil50 Not quite. You might've missed the last sentence:
image

**Pull request merged**